www/source/partials/docs/_glossary-keys.html.md.erb

```diff
@@ -6,13 +6,13 @@ Habitat has strong cryptography built into both Habitat Builder and the Supervis
 
 Every package and artifact in Habitat belongs to an [origin](/docs/glossary/#glossary-packages) and is cryptographically signed with that origin's private key.
 Habitat requires the public origin key for working with origin packages, both to produce the un-compiled package and to consume the compiled Habitat artifact, or `.hart`.
-On the production side, the Builder or Depot uses the  origin private key to transmit un-compiled packages transmitted between the workstation or Studio to the Builder or Depot. On the consumption side, the origin private key is required for the Supervisor to validate the compiled  artifacts.
+On the production side, Builder--both the service and the Depot--use the origin private key to transmit packages between your development environment to Builder. On the consumption side, the Supervisors require the origin private key to verify the compiled artifacts.
```

Comment:
Why not use a proper em dash (—)?

Reply:
I'd say they use the origin private key to **sign** artifacts, not *transmit* (and not packages, really). And again, the supervisor doesn't having anything to do with the verification requirement, that's a general feature of the process of installing an artifact.